index.html

```diff
@@ -10,951 +12,1033 @@
-
+					<div class="skip-to">
+						<a href="#maincontent" class="skip-link">
+							Skip to the main content
+						</a>
+						<span class="visuallyhidden">Focus on the first article by skipping the navigation.</span>
+					</div>
           <div id="navbarMenu" class="navbar-menu">
-            <div class="navbar-end">
+            <nav class="navbar-end">
               <span class="navbar-item">
-                <a class="button is-white is-outlined">
+                <button class="button is-white is-outlined">
                   <span class="icon">
                     <i class="fa fa-home"></i>
                   </span>
                   <span>Home</span>
-                </a>
+                </button>
               </span>
               <span class="navbar-item">
-                <a class="button is-white is-outlined">
+                <button class="button is-white is-outlined">
                   <span class="icon">
                     <i class="fab fa-superpowers"></i>
                   </span>
                   <span>Examples</span>
-                </a>
+                </button>
               </span>
               <span class="navbar-item">
-                <a class="button is-white is-outlined">
+                <button class="button is-white is-outlined">
                   <span class="icon">
                     <i class="fa fa-book"></i>
                   </span>
                   <span>Documentation</span>
-                </a>
+                </button>
               </span>
               <span class="navbar-item">
-                <a class="button is-white is-outlined">
+                <button class="button is-white is-outlined">
                   <span class="icon">
                     <i class="fab fa-github"></i>
                   </span>
                   <span>View Source</span>
-                </a>
+                </button>
               </span>
-            </div>
+            </nav>
           </div>
         </div>
       </div>
     </div>
 
-    <div class="hero-body">
+    <section class="hero-body">
       <div class="container has-text-centered">
         <div class="column is-6 is-offset-3">
-          <div class="title has-text-white">We have web accessibility in mind</div>
-          <div class="subtitle has-text-white">
+          <h1 class="title has-text-white">We have web accessibility in mind</h1>
+          <h2 id="subscribtion" class="subtitle has-text-white">
             Would you like to get the latest news?
-          </div>
-          <div class="box">
-            <div class="field is-grouped">
-              <p class="control is-expanded">
-                <input class="input">
-              </p>
-              <p class="control">
-                <a class="button is-info">Notify Me</a>
-              </p>
-            </div>
-          </div>
+					</h2>
+          <form aria-labelledby="subscribtion" class="box">
+            <fieldset class="field is-grouped">
```

Comment:
Could you please add also class `no-border` for this tag.

Reply:
for which tag? for form?